### PR TITLE
[TT-5624] Return error when user imports payload with x-tyk-api-gateway

### PR DESCRIPTION
**Pull request by furkansenharputlu**

### CODE REVIEW

File: gateway/api.go [hunk 1]
```diff
@@ -2796,6 +2796,11 @@ func (gw *Gateway) validateOAS(next http.HandlerFunc) http.HandlerFunc {
 			return
 		}
 
+		if strings.HasSuffix(r.URL.Path, "/import") && oasObj.GetTykExtension() != nil {
```
Comment (jeffy-mathew) [inline gateway/api.go:2799]: we can do this in `makeImportedOASTykAPI`

Comment (jeffy-mathew) [reply gateway/api.go:2799]: Makes sense.

**Review by jeffy-mathew — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4113/merge
Commit:  
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2447737990)

**Review by lghiur — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: dd757f295be645961c99f78814b2e03b7f0cabf8 Return error when user imports payload with x-tyk-api-gateway (#4113)
Triggered by: push (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2447912331)

**Pull request merged**